provider/datagen/src/transform/cldr/units/helpers.rs

```diff
@@ -421,22 +422,18 @@ pub(in crate::provider) fn is_scientific_number(s: &str) -> bool {
     !contains_alphabetic_chars(base) && !contains_alphabetic_chars(exponent)
 }
 
-/// Transforms a fractional number into byte numerators, byte denominators, and a sign.
+/// Converts a fractional number into its byte representation for numerator and denominator, along with its sign.
```

Comment:
consider moving this to the experimental crate, something like
```rust
impl IcuRatio {
  #[cfg(feature = "datagen")]
  pub fn to_parts(&self) -> (Vec<u8>, Vec<u8>, Sign) { ... }
}
```

Reply:
yes,  I want to have a struct that can be converted from and to IcuRatio to be implemented in Datagen